packages/nextjs-template/components/DendronTreeMenu.tsx

```diff
@@ -46,31 +48,28 @@ export default function DendronTreeMenu(
 
   // --- Verify
   // If no data, show spinner component
-  if (!verifyNoteData(props)) {
+  if (!props.tree || !noteActiveId || !props.noteIndex) {
```

Comment:
is it possible not to have `props.tree`? since we are passing it in at build time, shouldn't this prop be non-optional?
same with `noteIndex` and `noteActiveId`?